annotation/functions.scm

```diff
@@ -89,10 +89,10 @@
                (Evaluation (Predicate intr)
                      (Set  (Variable "$p1") (Variable "$p2")))))
          (run-query 
-            (Get 
+            (Bind 
```

Comment:
The function `(do-find-pathway/go-protein-interactors node)` only searches for String interactions, Biogrid interactions (of the Gene expressing the protein) needs to be included in the result.

Reply:
The input should only be a Gene? Tried to annotate a Protein "P05019" and got empty result